server/portal/apps/workspace/api/views.py

```diff
@@ -151,6 +141,54 @@ def get(self, request, *args, **kwargs):
             encoder=BaseTapisResultSerializer
         )
 
+    def select(self, client, request):
+        job_uuid = request.GET.get('job_uuid')
+        data = client.jobs.getJob(jobUuid=job_uuid)
+
+        return data
+
+    def listing(self, client, request):
+        limit = int(request.GET.get('limit', 10))
+        offset = int(request.GET.get('offset', 0))
+        portal_name = settings.PORTAL_NAMESPACE
+
+        data = client.jobs.getJobSearchList(
+            limit=limit,
+            startAfter=offset,
+            orderBy='lastUpdated(desc),name(asc)',
+            _tapis_query_parameters={'tags.contains': f'portalName: {portal_name}'},
+            select='allAttributes'
+        )
+
+        return data
+
+    def search(self, client, request):
+
+        query_string = request.GET.get('query_string')
+
+        limit = int(request.GET.get('limit', 10))
+        offset = int(request.GET.get('offset', 0))
+        portal_name = settings.PORTAL_NAMESPACE
+
+        sql_queries = [
+            f"(tags IN ('{portal_name}')) AND",
```

Comment:
Latest merge changed the tag to `f'portalName: {portal_name}'`
```suggestion
            f"(tags IN ('portalName: {portal_name}')) AND",
```